README.md

```diff
@@ -58,13 +47,16 @@ I tried to make the Vagrantfile as readable as possible but it's worth repeating
 |---|---|---|---|
 |8800|API|oms-core|8080|
 |8801|consumer|oms-profiles-module|8081|
+|---|API|oms-events|8082|
```

Comment:
no forwarding? useful if you want to test via curl (provided you snoop the token)... not mandatory though :)